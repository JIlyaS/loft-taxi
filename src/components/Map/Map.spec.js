import {Map} from './';
// import { cleanup, render } from '@testing-library/react';
import renderer from 'react-test-renderer';

describe('Map', () => {

  it('renders correctly snapshot', () => {
      const tree = renderer.create(<Map />).toJSON();
      expect(tree).toMatchSnapshot();
    });
  
  // не проходит из-за this.map.remove()
  // it('render correct', () => {
  //   const { queryByTestId } = render(<Map />);
  //   expect(queryByTestId('map')).toBeInTheDocument();
  // });
});
