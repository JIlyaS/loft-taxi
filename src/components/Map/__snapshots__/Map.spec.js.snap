// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Map renders correctly snapshot 1`] = `
<div
  className="map"
>
  <div
    className="map__component"
    data-testid="map"
  />
</div>
`;
